### DISCUSSION TITLE
Let you change the default create project directory in quick settings in the project manager

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [godotengine/godot-proposals#9511] [Editor]
change this directory
![image](https://github.com/godotengine/godot-proposals/assets/88291316/e521aba0-a1db-443a-9538-f12e415a9123)

with a setting here
![image](https://github.com/godotengine/godot-proposals/assets/88291316/3b649eb7-639e-4453-8e7c-3170cc1c8981)

or somewhere else. Its not important though

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/6070
